File: cpp/fib.cc
/*
 * Name: fib
 * Description: Paul's fibonacci
 */
#include <iostream>

using namespace std;

int main(void)
{
  int a,b,c;

  a=1;
  b=0;
  c=0;

  for(int x=1; x <= 10; x++)
  {
    c = a + b;
    b = a;
    a = c;
    cout << "c: " << c << endl;
  } 

  return 1;
}

File: php/fib.php
<?php
  /**
   * Paul's fibonacci
   **/
  $a=1;
  $b=0;
  $c=0;
  for($f=1;$f<10;$f++)
  {
     $c = $a + $b;
     $b = $a; // prior
     $a = $c; // current
     echo "fib: ".$c."\n";
  }

  echo "\n";
?>
